app/build.gradle

```diff
@@ -83,6 +82,7 @@ dependencies {
     implementation platform('com.google.firebase:firebase-bom:31.2.3')
     implementation 'com.google.firebase:firebase-auth-ktx'
     implementation 'com.google.firebase:firebase-firestore-ktx'
+    implementation 'com.google.firebase:firebase-database-ktx'
```

Comment:
We don't need firebase anymore, we now have firestore 🤩

Reply:
I am not familiar with the database stuff so what lines should I remove?
Should I juste keep line 84?

Reply:
Remove just line 85 and keep the rest